packages/web/pages/pool/[id].tsx

```diff
@@ -289,6 +290,16 @@ const Pool: FunctionComponent = observer(() => {
   const levelCta = bondLiquidityConfig?.calculateBondLevel(bondableDurations);
   const level2Disabled = bondableDurations.length === 0;
 
+  const highestAPRDailyPeriodicRate =
+    bondableDurations[bondableDurations?.length - 1]?.aggregateApr
```

Comment:
Technically, we should be subtracting the swap fee APR from the aggregate since users do earn swap fees on pool shares. Additional incentives are given (from a gauge) when the user locks the shares in a lock. LMK if you want more of a rundown of how the incentive mech works.

Reply:
But aren't swap fees part of the reward? I thought this number was the result of calculating the potential income from bonding the available value.